topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -268,6 +270,14 @@ For this tutorial, we will use the default "peer" authentication, so we need to
 > 1. Create and edit `group_vars/galaxyservers.yml` and add some variables to configure PostgreSQL:
 >
 >    ```yaml
+>    ---
+>    # python3 support
+>    galaxy_venv_python: 3
+>    pip_virtualenv_command: /usr/bin/python3 -m virtualenv # usegalaxy_eu.certbot, usegalaxy_eu.tiaas2, galaxyproject.galaxy
+>    certbot_virtualenv_package_name: python3-virtualenv    # usegalaxy_eu.certbot
```

Comment:
The `virtualenv` package in Ubuntu 18.04 depends on `python3-virtualenv` and can be more easily invoked than `/usr/bin/python3 -m virtualenv`

Reply:
Ok, since we're now installing that, that's great. We should test this again without the `python3-virtualenv`.

Reply:
`python3-virtualenv` is installed anyway as dependency of `virtualenv`, so it should be fine.